src/main/scala/cromwell/engine/WorkflowActor.scala

```diff
@@ -54,31 +52,58 @@ class WorkflowActor private(binding: WdlBinding, actualInputs: Map[FullyQualifie
   // That which started the workflow, a role which has not yet been defined more specifically.
   private var _primeMover: Option[ActorRef] = None
 
-  private def isStarted: Boolean = _primeMover.isDefined
+  override def receive = notStarted
 
-  private def primeMover = _primeMover.get
+  private var maybeSymbolStore: Option[SymbolStore] = None
+  private var maybeExecutionStatusStore: Option[ExecutionStatusStore] = None
+  private var maybeBackend: Option[Backend] = None
+  private val log = Logging(context.system, this)
 
-  override def receive: Receive = {
+
+  private def notStarted: Receive = {
     LoggingReceive {
 
-      case Start =>
+      case Start(backend) =>
         _primeMover = Option(sender())
-        val executionStatusStore = new ExecutionStatusStore(binding)
-        val symbolStore = new SymbolStore(binding, actualInputs)
+        maybeExecutionStatusStore = Option(new ExecutionStatusStore(binding))
```

Comment:
discussed in person, will stay as it is due to the particular circumstances here